content/docs/use-cases/ml-model-registries.md

```diff
@@ -0,0 +1,54 @@
+# Machine Learning Model Registries
+
+An **ML model registry** (or _model catalog_) works as a central repository
```

Comment:
this is not only about being in between development and apps (another way to say - deployment, or production, etc) .... this is about discoverability - teams need this to see what models are available, who was building them, how, etc ...

Reply:
Updated intro. PTAL when you can

Reply:
Looks good!

Second part `DVC supports it ` ... can we mention that DVC kinda enables it on top of Git ecosystem ? so, it's the same but a bit more explicit. Or it can be something like ... DVC runs on top of Git to .... wdyt?

Reply:
I'm not sure we want to mention Git in the intro paragraph or how to do that without getting technical. Let me try... ⌛

Reply:
yep, maybe (Git ecosystem) would be enough? not sure ... I see your concern, but when we talk about DVC in other use cases we mention Git, so probably it's fine

Reply:
What about "DVC enables these features on the Git ecosystem, reducing friction between development and operations." ?

Reply:
or keep "..., leveraging existing development workflows."

Reply:
a bit too fancy :) if we can do the same a bit more explicit that would be great ... something like "DVC is a CLI tool that works on top of Git to get you this features", wdyt?

Reply:
OK. I thought we wanted keep the whole term "Git ecosystem" for SEO. Much easier to phrase as just "Git". Done:

> DVC works on top of Git to enable these capabilities on your existing ecosystem.